spread.yaml

```diff
@@ -22,6 +22,18 @@ prepare: |
     echo "Spread is running as $(id)"
     [ "$REUSE_PROJECT" != 1 ] || exit 0
     ./spread-tests/spread-prepare.sh
+    # trusty support is under development right now
+    # we special-case the release until we have officially landed
+    if ["$(lsb_release -sc)" == "trusty"]
```

Comment:
lsb_release is not preinstalled everywhere, please move this to spread-prepare.sh where we already have all the information `release_ID` and `release_VERSION_ID`. Please do that before we `apt-get install snapd`.